smurfs/src/actions/index.js

```diff
@@ -1,8 +1,14 @@
+import axios from "axios";
+
 /* 
   Action Types Go Here!
   Be sure to export each action type so you can pull it into your reducer
 */
-
+export const FETCH_SMURF_START = 'FETCH_SMURF_START';
+export const FETCH_SMURF_SUCCESS = 'FETCH_SMURF_SUCCESS';
+export const FETCH_SMURF_FAILURE = 'FETCH_SMURF_FAILURE';
+export const SMURF_ADD = 'SMURF_ADD'
+export const SMURF_DELETE = 'SMURF_DELETE'
 /*
```

Comment:
you will want to make separate actionTypes for each of the states in the finite machine model and not reuse the same ones. this will ensure that the modifications to state happen the exact same each time. so insead of reusing the fetch_smurf_success and failure you would create a smurf_add_success / _failure and a surf_delete_success / _failure actionTypes as well and pass those to the reducer